dev-resources/less/grid.less

```diff
@@ -27,6 +27,11 @@
   background-color: @hento-korostusvari;
 }
 
+.otsikko-ei-taustaa {
```

Comment:
Mietin onko meillä syy, että juuri tässä raportissa halutaan raportti ilman taustaa, vai pitäskö mennä sillä että se poistetaan kaikista, jotta säilyy yhtenäinen look and feel?

Reply:
tähän oli syy että tehtiin tyylitön välitaulukko raporttiin